django_tasks/backends/database/management/commands/db_worker.py

```diff
@@ -94,19 +94,16 @@ def start(self) -> None:
                 if task_result is not None:
                     self.run_task(task_result)
 
+                else:
+                    time.sleep(self.interval)
```

Comment:
I'm not sure moving the `sleep` is the right approach. Moving it here sets `running_task` longer than necessary.

Keeping it after the loop probably makes sense. But perhaps it should be `if self.running and task_result is None`?

Reply:
That makes sense. I've changed the `if` condition, and did a quick test, it's working as expected.